sdk/src/api/mod.rs

```diff
@@ -143,7 +143,6 @@ pub type Result<T> = std::result::Result<T, crate::Error>;
 type Waiter = (watch::Sender<Option<WaitFor>>, watch::Receiver<Option<WaitFor>>);
 
 const SUPPORTED_VERSIONS: (&str, &str) = (">=1.0.0, <3.0.0", "20230701.55918b7c");
-const REVISION_SUPPORTED_SERVER_VERSION: Version = Version::new(1, 2, 0);
```

Comment:
Don't the changes in this PR mean the SDK can only communicate with `v1.2+` now? If so, we should update `SUPPORTED_VERSIONS` accordingly.

Reply:
yes, though over http, it requires 1.5.4>=